src/main/scala/nl.knaw.dans.easy.sword2/DepositHandler.scala

```diff
@@ -289,9 +289,9 @@ object DepositHandler {
     }
 
     def extract(file: File, outputPath: String): Unit = {
-      new ZipFile(file.getPath) {
-        setFileNameCharset(StandardCharsets.UTF_8.name)
-      }.extractAll(outputPath)
+      val f = better.files.File(file.getAbsolutePath)
```

Comment:
use an import statement above instead of `better.files.File`. Or even better

```scala
import better.files._

file.toScala // transforms Java IO File to better-files File
```

Same for the `String outputPath` below: use `outputPath.toFile`.

For reference, see https://github.com/pathikrit/better-files/blob/b4ad1a095eddf8bbec592812bd38bb4a335bb712/core/src/main/scala/better/files/Implicits.scala#L31-L48